deploy/src/main/resources/production-marathon-testing/marathon.env

```diff
@@ -0,0 +1,19 @@
+# The following properties MUST be configured before the Job Service Prerequisites can be deployed.
+
+export CAF_RABBITMQ_HOST=NOT_SET_CAF_RABBITMQ_HOST
+export CAF_RABBITMQ_PORT=NOT_SET_CAF_RABBITMQ_PORT
+export CAF_RABBITMQ_PASSWORD=NOT_SET_CAF_RABBITMQ_PASSWORD
+export CAF_RABBITMQ_USERNAME=NOT_SET_CAF_RABBITMQ_USERNAME
+
+export CAF_WORKER_GLOBFILTER_INPUT_QUEUE=NOT_SET_CAF_WORKER_GLOBFILTER_INPUT_QUEUE
+export CAF_BATCH_WORKER_ERROR_QUEUE=NOT_SET_CAF_BATCH_WORKER_ERROR_QUEUE
+export CAF_GLOB_WORKER_BINARY_DATA_INPUT_FOLDER=NOT_SET_CAF_GLOB_WORKER_BINARY_DATA_INPUT_FOLDER
+
+export CAF_WORKER_LANGDETECT_INPUT_QUEUE=NOT_SET_CAF_WORKER_LANGDETECT_INPUT_QUEUE
+export CAF_WORKER_LANGDETECT_OUTPUT_QUEUE=NO_SET_CAF_WORKER_LANGDETECT_OUTPUT_QUEUE
+export CAF_LANG_DETECT_WORKER_OUTPUT_FOLDER=NOT_SET_CAF_LANG_DETECT_WORKER_OUTPUT_FOLDER
+
+export JOB_SERVICE_DEMO_INPUT_DIR=NOT_SET_JOB_SERVICE_DEMO_INPUT_DIR
+export JOB_SERVICE_DEMO_OUTPUT_DIR=NOT_SET_JOB_SERVICE_DEMO_OUTPUT_DIR
+
+export WORKER_STORAGE_HOST_DATA_DIRECTORY=NOT_SET_WORKER_STORAGE_HOST_DATA_DIRECTORY
```

Comment:
I feel like this should be prefixed with `CAF_`.

Reply:
Fixed